app.js

```diff
@@ -1,8 +1,125 @@
 // import functions and grab DOM elements
 
+import { renderPastPoll } from './render.js';
+const aVoteUp = document.getElementById('option-a-add');
+const bVoteUp = document.getElementById('option-b-add');
+const aVoteDown = document.getElementById('option-a-sub');
+const bVoteDown = document.getElementById('option-b-sub');
+const endPoll = document.getElementById('end-poll');
+
+//const questionInput = document.querySelector('.question-area');
+//const option1Input = document.querySelector('.question-1');
+//const option2Input = document.querySelector('.question-2');
+
+const questionLabel = document.getElementById('question-label');
+
+const option1Label = document.getElementById('option-1-label');
+const option1Votes = document.getElementById('option-1-votes');
+const option2Label = document.getElementById('option-2-label');
+const option2Votes = document.getElementById('option-2-votes');
+const form = document.querySelector('form');
+const pastPolls = document.querySelector('.past-polls');
+
+
 // let state
+let question = '';
+let aVotes = 0;
+let bVotes = 0;
+let aTitle = '';
+let bTitle = '';
+
+const pastPollsArray = [];
+
+// set event listeners
+aVoteUp.addEventListener('click', () => {
+    aVotes++;
+    displayCurrentPoll();
+});
+
+bVoteUp.addEventListener('click', () => {
+    bVotes++;
+    option2Votes.textContent = bVotes;
+});
+
+aVoteDown.addEventListener('click', () => {
+    aVotes--;
+    displayCurrentPoll();
+});
+
+bVoteDown.addEventListener('click', () => {
+    bVotes--;
+    displayCurrentPoll();
+});
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const data = new FormData(form);
+
+    question = data.get('question-area');
+    aTitle = data.get('question-1');
+    bTitle = data.get('question-2');
+
+    displayCurrentPoll();
+
+});
+
+endPoll.addEventListener('click', () => {
+    form.reset();
```

Comment:
I'd move this form reset up into your form event listener so that the last thing you do after grabbing data and displaying it is clear out the form. It's nice to just handle all of that within the form event listener